src/main/java/frc/robot/subsystem/Claw.java

```diff
@@ -0,0 +1,77 @@
+package frc.robot.subsystem;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+// Pneumatics HUB and Controller stuff??
+
+enum ClawStates {
+    IDLE,
+    OUTPUTING,
+    INTAKING,
+    HOLDING
+}
+
+public class Claw {
+    // Solenoids for claw open and close
+    private Solenoid solenoidLeft;
+    private Solenoid solenoidRight;
+    
+    // NEO 550 motors for wheel spinning
+    private CANSparkMax motorLeft;
+    private CANSparkMax motorRight;
+
+    private ClawStates state;
+
+    
+    public Claw() {
+        // Initialize solenoids
+        solenoidLeft = new Solenoid(PneumaticsModuleType.CTREPCM, Constants.Claw.LEFT_PNEUMATICS_ID); // NOT ACCURATE
+        solenoidRight = new Solenoid(PneumaticsModuleType.CTREPCM, Constants.Claw.RIGHT_PNEUMATICS_ID); // NOT ACCURATE
+        
+        // Initialize NEO 550 motors
+        motorLeft = new CANSparkMax(Constants.Claw.LEFT_MOTOR_ID, MotorType.kBrushless); // NOT ACCURATE
```

Comment:
I recommend having one motor follow the other (motorLeft.follow(motorRight)). And then inverting one